contracts/vaults/RibbonThetaVault.sol

```diff
@@ -390,8 +390,18 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
             expiry = getNextFriday(IOtoken(oldOption).expiryTimestamp());
         }
 
+        IStrikeSelection strikeSelection = IStrikeSelection(strikeSelection);
+
         (uint256 strikePrice, uint256 delta) =
-            IStrikeSelection(strikeSelection).getStrikePrice(expiry, isPut);
+            strikeOverride.lastStrikeOverride == round
+                ? (
+                    strikeOverride.overriddenStrikePrice,
+                    strikeSelection.delta()
```

Comment:
in the strike override we assume the strike price will be 0.1 delta, or whatever the delta is in strike selection contract. can we make this assumption or should we add an additional param to setStrikePrice that sets some `overrideDelta` variable.

Reply:
Seems reasonable to just return the `strikeSelection.delta()`. Don't think the delta matters in that scenario